src/modelstats.jl

```diff
@@ -3,47 +3,18 @@ export dic
 #################### Posterior Statistics ####################
 
 """
-    dic(chain::Chains, logpdf::Function) -> (DIC, pD)
+    dic(chain::Chains)
 
-Compute the deviance information criterion (DIC).
-(Smaller is better)
+Computes Deviance Information Criterion (DIC).
 
-Note: DIC assumes that the posterior distribution is approx. multivariate Gaussian and tends to select overfitted models.
-
-## Returns:
-* `DIC`: The calculated deviance information criterion
-* `pD`: The effective number of parameters
-
-## Usage:
-
-```
-chn ... # sampling results
-lpfun = function f(chain::Chains) # function to compute the logpdf values
-    niter, nparams, nchains = size(chain)
-    lp = zeros(niter + nchains) # resulting logpdf values
-    for i = 1:nparams
-        lp += map(p -> logpdf( ... , x), Array(chain[:,i,:]))
-    end
-    return lp
-end
-
-DIC, pD = dic(chn, lpfun)
-```
+# Arguments
+* `chain::Chains: an MCMC `Chain` object containing posterior log likelihood values
+* `loglik::Symbol`: variable name associated with posterior log likelihood values
```

Comment:
The convention in MCMCChains (and [in the Julia guidelines](https://docs.julialang.org/en/v1/manual/documentation/)) is to not provide a list of arguments but to integrate them in the description if possible. Could you change this?

Reply:
I updated the docs. Sorry. I was not aware of the Julia guidelines.